util/strings.go

```diff
@@ -19,3 +20,12 @@ func RandomString(length int) string {
 	s := sb.String()
 	return s
 }
+
+// ApplyColor applies a color to a text to use in Kodi skin
+func ApplyColor(text string, color string) (coloredText string) {
+	coloredText = text
+	if color != "none" {
```

Comment:
```suggestion
	if color != "" && color != "none" {
```

Let's also check for empty color

Reply:
good catch, thanks. commited.